com.sap.cloud.lm.sl.persistence/src/main/java/com/sap/cloud/lm/sl/persistence/services/AbstractFileService.java

```diff
@@ -403,6 +405,34 @@ public Integer execute(Connection connection) throws SQLException {
         return listFiles(null, namespace);
     }
 
+    public List<FileEntry> listByModificationTime(final Date modificationTime) throws FileStorageException {
+        SqlExecutor<List<FileEntry>> executor = new FileServiceSqlExecutor<List<FileEntry>>();
```

Comment:
I beleive, Suni did some refactoring here? Have you adopted it in your code?

Reply:
After rebase it will be in place.